packages/web-components/fast-foundation/src/progress/progress.template.ts

```diff
@@ -27,7 +27,7 @@ export const ProgressTemplate: ViewTemplate<BaseProgress> = html`
             `
         )}
         ${when(
-            x => !x.value,
+            x => typeof x.value !== "number" || x.value === null,
```

Comment:
Same situation here:

```suggestion
            x => typeof x.value !== "number",
```